layers/raw.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2012 Google, Inc. All rights reserved.
+// Copyright 2009-2011 Andreas Krennmair. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"fmt"
+	"github.com/google/gopacket"
+)
+
+// RawIP (DLT_RAW) contains no header and we start with the IP header
+type Raw struct {
```

Comment:
It would behave similar to how wireshark is showing the stacking for raw frames, maybe if the encapsulation layer gains a way to write pcap files having a separate interface for the encapsulating layer helps?
Other than that we don't really need to add this layer right now.